infrastructure/pipeline_stack/elastic_cluster.tf

```diff
@@ -18,7 +18,7 @@ resource "ec_deployment" "content_cluster" {
     autoscale = "false"
     topology {
       id         = "hot_content"
-      size       = "2g"
+      size       = "1g"
```

Comment:
The cluster was already this size - I'm not sure why we had `2g` in tf, I suspect we intended to scale it up then never did.